Leanplum-SDK/Classes/Managers/Networking/LPRequestFactory.h

```diff
@@ -3,7 +3,34 @@
 
 @interface LPRequestFactory : NSObject
 
-+ (id<LPRequesting>)get:(NSString *)apiMethod params:(NSDictionary *)params;
-+ (id<LPRequesting>)post:(NSString *)apiMethod params:(NSDictionary *)params;
+//+ (id<LPRequesting>)get:(NSString *)apiMethod params:(NSDictionary *)params;
+//+ (id<LPRequesting>)post:(NSString *)apiMethod params:(NSDictionary *)params;
+
++ (id<LPRequesting>)apiMethodStartWithParams:(NSDictionary *)params;
```

Comment:
I think we can remove the `apiMethod` part of this. I understand it is very descriptive, but the names are becoming prohibitively long and I think it's pretty clear what they do.